testing.rst

```diff
@@ -513,14 +513,14 @@ returns a ``Crawler`` instance.
 The full signature of the ``request()`` method is::
 
     request(
-        $method,
-        $uri,
+        string $method,
+        string $uri,
         array $parameters = [],
         array $files = [],
         array $server = [],
-        $content = null,
-        $changeHistory = true
-    )
+        string $content = null,
```

Comment:
```suggestion
        ?string $content = null,
```
?

Reply:
It makes more sense with '?' cause it's nullable but i've followed the exact declaration : 
**public function request(string $method, string $uri, array $parameters = [], array $files = [], array $server = [], string $content = null, bool $changeHistory = true)**

Reply:
I agree with Mohamed. Thanks!